CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Fixed log levels in JNI layer (#4204).
 * Fixed a bug in encryption (#4128).
 * Fixed "Read-only file system" exception when compacting Realm file on external storage (#4140).
+* Fixed element type checking in `DynamicRealmObject#setList()` (#4252).
```

Comment:
No need to prefix with "Fixed" when it is under the section "Bug fixes". It is redundant.